Jenkinsfile

```diff
@@ -250,15 +272,17 @@ pipeline {
           steps {
             sh 'export'
             sh 'make distclean; rm -rf .ros; rm -rf .gazebo'
-            sh 'git fetch --tags'
-            sh 'make posix_sitl_default'
-            sh 'make posix_sitl_default sitl_gazebo'
-            sh './test/rostest_px4_run.sh mavros_posix_test_mission.test mission:=vtol_new_1 vehicle:=standard_vtol'
-            sh './Tools/ecl_ekf/process_logdata_ekf.py `find . -name *.ulg -print -quit`'
+            unstash 'px4_sitl_package'
+            sh 'unzip build/posix_sitl_default/px4-posix_sitl_default*.zip'
+            sh 'mv px4-posix_sitl_default*/bin/px4 px4-posix_sitl_default*/share/px4/'
```

Comment:
This is a result of cpack using different bin and share dirs. It needs to be moved for ROS.